src/main/resources/db/changelog/changes/v1.0.0/sql/derived-tables/create-view-user-details.sql

```diff
@@ -29,7 +29,54 @@ SELECT
         WHEN '004' THEN 'phone'
         WHEN '005' THEN 'mobile'
         ELSE 'unknown'
-    END AS preferred_contact_type
+    END AS preferred_contact_type,
```

Comment:
I know this code is kind of ugly but could not figure out any other way to make it run. (Tried other numerous ways but everything failed :( )